### PR TITLE
feat(credentials): Holder-Integrate Credentials verification -> Added VC body validation on verification & tests [DEV-1219]

### PR DESCRIPTION
**Pull request by Eengineer1**
- Credential payload - JWT payload equality extension.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eengineer1/did-jwt-vc",
```
Comment (ankurdotb) [inline package.json:2]: Swap to cheqd

File: package.json [hunk 2]
```diff
@@ -27,8 +27,9 @@
   ],
   "license": "ISC",
```
Comment (ankurdotb) [inline package.json:28]: Also swap the license to ours (Apache 2.0)

**Review by ankurdotb — CHANGES_REQUESTED** [review]
I'm gonna reset this PR to draft, so that we get the chance to incorporate linting etc workflows first.

I also wouldn't mind a quick review from @askolesov. We've built some validation logic for DIDs in the ledger side, perhaps we could use the same here? Or at least they shouldn't fundamentally be too far away.

**Comment by ankurdotb** [comment]
@Eengineer1 Is this one deprecated?

**Comment by Eengineer1** [comment]
@ankurdotb Yes, we had switched it to draft, but let's close, if you agree as well.

**Pull request closed without merging**